src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -115,20 +114,18 @@
                                                token
                                                token-balance))
             fiat-limit                    (.toFixed (* token-balance conversion-rate) 2)
-            current-limit                 #(if @crypto-currency? crypto-limit fiat-limit)
-            routes-can-be-fetched?        (and (= nav-current-screen-id current-screen-id)
-                                               (not
-                                                (or (empty? (controlled-input/input-value input-state))
-                                                    (<= (controlled-input/numeric-value input-state) 0)
-                                                    (> (controlled-input/numeric-value input-state)
-                                                       (current-limit)))))
+            current-limit                 (if @crypto-currency? crypto-limit fiat-limit)
+            valid-input?                  (not (or (empty? (controlled-input/input-value input-state))
```

Comment:
empty? Won't detect whitespace
Better to use clojure.string/blank?

Reply:
@ibrkhalil, fixed)